client/Components/App.test.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { shallow } from "enzyme";
-import App from "./App.jsx";
+import App from "./App";
```

Comment:
Would it have been better to leave the .jsx? I'm not completely sure if it matters too much.

Reply:
I don't even remember changing thats but will fix asap.